src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--mon
```

Comment:
Test crying😥 Because You don't must added special div in modificator. In my opinion, this modificator can be added in  element
```suggestion
    <div class="calendar calendar__start-day--mon
```

Reply:
or  this variant
```suggestion
     <div class="calendar calendar--month-length_31">
        <div class="calendar__day calendar__day--1 calendar__day--start-day_sun"></div>
```

Reply:
In my opinion, it's not an element) it's a modifier for block calendar. This style is called 'Two Dashes'. Also in [checklist](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md)point 2 we have a necessary example. )